stripper/ze_crazy_escape_v5_5_1.cfg

```diff
@@ -1,3 +1,17 @@
+;Block ZM multiple ZM nuke avoidance spots by surfing on lava
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "nrk3_final_win"
+	}
+	insert:
+	{
+		"OnTrigger" "playerRunScriptCodeforeach(k,_ in{SetHealth=0}){if(self.GetTeam()==2&&self.GetHealth()>0)EntFireByHandle(self, k,(0).tostring(),0,null,null)}11"
```

Comment:
It also won't trigger repeat killer, so delay this to 1.1 seconds so nrk3_final_hurt_all is always responsible for killing most zombies.